### DISCUSSION TITLE
Unattended installation

### DISCUSSION THREAD

**Discussion by kakulukia** [starship/starship#1820] [Q&A] [answered]
Looking at the install.sh script i see options like -y to just install starship.
Is there a way to use that with 

    curl -fsSL https://starship.rs/install.sh | bash

that line to keep it silent?

**Comment by davidkna** [comment, marked as answer]
I think you need `curl -fsSL https://starship.rs/install.sh | bash -s -- -y`

**Reply by kakulukia** [reply]
awesome - woks!
